src/releasers/helm.ts

```diff
@@ -0,0 +1,144 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {ReleasePR, ReleaseCandidate} from '../release-pr';
+
+import {ConventionalCommits} from '../conventional-commits';
+import {GitHub, GitHubTag, GitHubFileContents} from '../github';
+import {checkpoint, CheckpointType} from '../util/checkpoint';
+import {Update} from '../updaters/update';
+import {Commit} from '../graphql-to-commits';
+
+// Generic
+import {Changelog} from '../updaters/changelog';
+import * as yaml from 'js-yaml';
+// helm
+import {ChartYaml} from '../updaters/helm/chart-yaml';
+
+export class Helm extends ReleasePR {
+  static releaserName = 'helm';
+
+  protected async _run(): Promise<number | undefined> {
+    // Make an effort to populate packageName from the contents of
+    // the package.json, rather than forcing this to be set:
+    const contents: GitHubFileContents = await this.gh.getFileContents(
+      this.addPath('Chart.yaml')
+    );
+    const file = yaml.load(contents.parsedContent, {json: true});
+    if (file === null || file === undefined) {
+      return undefined;
+    }
+    const pkg = JSON.parse(JSON.stringify(file));
+    if (pkg.name) {
+      this.packageName = pkg.name;
+      // we've rewritten the package name, recalculate the package prefix
+      this.packagePrefix = this.coercePackagePrefix(pkg.name);
+    }
+
+    const latestTag: GitHubTag | undefined = await this.gh.latestTag(
+      this.monorepoTags ? `${this.packagePrefix}-` : undefined
+    );
+    const commits: Commit[] = await this.commits({
+      sha: latestTag ? latestTag.sha : undefined,
+      path: this.path,
+    });
+
+    const cc = new ConventionalCommits({
+      commits,
+      githubRepoUrl: this.repoUrl,
+      bumpMinorPreMajor: this.bumpMinorPreMajor,
+      changelogSections: this.changelogSections,
+    });
+    const candidate: ReleaseCandidate = await this.coerceReleaseCandidate(
+      cc,
+      latestTag
+    );
+    const changelogEntry: string = await cc.generateChangelogEntry({
```

Comment:
I'll trust you're the expert where the actual Helm update process is concerned 😄 

I've just asked that folks own the first few bugs in their releaser, as other people adopt it.